pulsar-client-reactive-adapter/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
 	testImplementation libs.mockito.inline
 
 	intTestImplementation project(':pulsar-client-reactive-producer-cache-caffeine')
+	intTestImplementation project(path: ':pulsar-client-reactive-producer-cache-caffeine-shaded', configuration: 'shadow')
```

Comment:
So this is interesting, bringing in both shaded and non-shaded. This is to allow the intTest to use both shaded and non-shaded to make sure all is working well in both.